### PR TITLE
Documentation update for v0.0.16

### PR DESCRIPTION
**Pull request by sdirosa**
- Update all documentation referring to console to mention instead TrackerConsole.
- Added how-to guide on how to configure logging with TrackerConsole.
- Updated how-to guide on how to configure RootLocationContextFromURLPlugin to use the new configurations.
- Updated default factories method names to the latest versions, prefixed with the browser name.
- Updated all Trackers and Tracker factories signatures with the new plugin configuration flags.
- Added an admonition about RootLocationContext being required next to disabling the plugin.

### CODE REVIEW

**Comment by sdirosa** [comment]
@ivarpruijn I've updated the documentation to reflect all the changes introduces in https://github.com/objectiv/objectiv-analytics/pull/536. I've also added a new how-to about logging configuration for all trackers.

File: docs/docs/tracking/angular/how-to-guides/configuring-logging.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 5
+---
+
+# Configuring logging
+
+All Trackers, Transports, Queues and Plugins log to TrackerConsole global singleton.
```
Comment (ivarpruijn) [inline docs/docs/tracking/angular/how-to-guides/configuring-logging.md:7]:
```suggestion
All Trackers, Transports, Queues and Plugins log to a TrackerConsole global singleton.
```

File: docs/docs/tracking/angular/how-to-guides/configuring-logging.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+## What is TrackerConsole
+TrackerConsole is a simplified wrapper implementation of node.js and browsers Console.
```
Comment (ivarpruijn) [inline docs/docs/tracking/angular/how-to-guides/configuring-logging.md:10]:
```suggestion
TrackerConsole is a simplified wrapper implementation of node.js and browsers' Console.
```

File: docs/docs/tracking/browser/how-to-guides/configuring-logging.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 4
+---
+
+# Configuring logging
+
+All Trackers, Transports, Queues and Plugins log to TrackerConsole global singleton.
```
Comment (ivarpruijn) [inline docs/docs/tracking/browser/how-to-guides/configuring-logging.md:7]:
```suggestion
All Trackers, Transports, Queues and Plugins log to a TrackerConsole global singleton.
```

File: docs/docs/tracking/browser/how-to-guides/configuring-logging.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+## What is TrackerConsole
+TrackerConsole is a simplified wrapper implementation of node.js and browsers Console.
```
Comment (ivarpruijn) [inline docs/docs/tracking/browser/how-to-guides/configuring-logging.md:10]:
```suggestion
TrackerConsole is a simplified wrapper implementation of node.js and browser's Console.
```

File: docs/docs/tracking/react/how-to-guides/configuring-logging.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 6
+---
+
+# Configuring logging
+
+All Trackers, Transports, Queues and Plugins log to TrackerConsole global singleton.
```
Comment (ivarpruijn) [inline docs/docs/tracking/react/how-to-guides/configuring-logging.md:7]:
```suggestion
All Trackers, Transports, Queues and Plugins log to a TrackerConsole global singleton.
```

File: docs/docs/tracking/react/how-to-guides/configuring-logging.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+## What is TrackerConsole
+TrackerConsole is a simplified wrapper implementation of node.js and browsers Console.
```
Comment (ivarpruijn) [inline docs/docs/tracking/react/how-to-guides/configuring-logging.md:10]:
```suggestion
TrackerConsole is a simplified wrapper implementation of node.js and browser's Console.
```

**Review by ivarpruijn — COMMENTED** [review]
Nice update. Only some super minor suggestions.

File: docs/docs/tracking/angular/how-to-guides/configuring-logging.md (hunk 1 above)
Comment (sdirosa) [reply docs/docs/tracking/angular/how-to-guides/configuring-logging.md:7]: Why a? There can't be more than one. It's a singleton :)
Maybe `the`?

Comment (ivarpruijn) [reply docs/docs/tracking/angular/how-to-guides/configuring-logging.md:7]: Very true :). Then I think `the` works best. Without anything it's a bit weird to read.

Comment (ivarpruijn) [reply docs/docs/tracking/angular/how-to-guides/configuring-logging.md:7]: Updated the suggestion, will also do for the other ones.

Comment (sdirosa) [reply docs/docs/tracking/angular/how-to-guides/configuring-logging.md:7]: Noice

Comment (sdirosa) [reply docs/docs/tracking/angular/how-to-guides/configuring-logging.md]: For some reason I cannot commit these changes, I'll fix them manually

**Pull request merged**